examples_server/examples/data_table/large_table_column_reordering/data_table.js

```diff
@@ -0,0 +1,125 @@
+Backdraft.app("TableExample", {
+
+  plugins : [ "DataTable"],
```

Comment:
Good point, I copied this from the other examples.. which were all made back before the styleguide.  I'll see at making a pass across all the examples in backdraft!